ISICkonto/Flow/ViewControllers/LoginViewController.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  ViewController.swift
+//  ISICkonto
+//
+//  Created by Vendula Švastalová on 07/02/2019.
+//  Copyright © 2019 Vendula Švastalová. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class LogInViewController: UIViewController {
+    
+    private var logo: UIImageView!
+    private var usernameTF: FormField!
+    private var passwordTF: FormField!
+    private var loginButton: UIButton!
+    
+    var onLoginSuccess: (String) -> () = { _ in }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupUI()
+        createConstraints()
+    }
+    
+    func setupUI() {
+        self.view.backgroundColor = UIColor(red: 93/255, green: 149/255, blue: 170/255, alpha: 1)
+        
+        logo = UIImageView()
+        usernameTF = FormField()
+        passwordTF = FormField()
+        loginButton = UIButton()
+        
+        passwordTF.textContentType = UITextContentType.password
+        passwordTF.isSecureTextEntry = true
+        
+        usernameTF.textContentType = UITextContentType.username
+        
+        usernameTF.placeholder = "username".localized
+        passwordTF.placeholder = "password".localized
+        loginButton.setTitle("Log In".localized, for: .normal)
```

Comment:
bylo by hezčí nějak logicky uskupit nastavování jednotlivých viewček, tady je to jedno přes druhé a není to moc čitelné

Reply:
taky to ted vidim, chtela jsem to delat poporade, inicializace, a pak nejaky akce s tim, a uz tim to vypada dost osklive, takze proste asi kazda komponenta k sobe